api-contract/src/main/java/uk/gov/hmcts/fees2/register/api/contract/request/FixedFeeDto.java

```diff
@@ -14,8 +14,8 @@
 public class FixedFeeDto extends FeeDto{
 
     @Builder(builderMethodName = "fixedFeeDtoWith")
-    public FixedFeeDto(String code, String newCode, FeeVersionDto version, String jurisdiction1, String jurisdiction2, String service, String channel, String event, String applicantType, boolean unspecifiedClaimAmount, String keyword) {
-        super(code, newCode, version, jurisdiction1, jurisdiction2, service, channel, event, applicantType, unspecifiedClaimAmount, keyword);
+    public FixedFeeDto(String code, String newCode, FeeVersionDto version, String jurisdiction1, String jurisdiction2, String service, String channel, String event, String applicantType, boolean unspecifiedClaimAmount, String keyword, String reasonForUpdate) {
+        super(code, newCode, version, jurisdiction1, jurisdiction2, service, channel, event, applicantType, unspecifiedClaimAmount, keyword, reasonForUpdate);
```

Comment:
What is the requirement to add 'reasonForUpdate' in as attribute in FeeDTO Model ?

Reply:
This also updated but forgot to push, did that along with Encode